Source/Definition/Method/TyphoonMethod.m

```diff
@@ -90,9 +90,10 @@ - (NSUInteger)hash
 - (NSUInteger)calculateHash
 {
     NSUInteger hash = (NSUInteger) sel_getName(_selector);
+    NSUInteger magicNumber = 5;
 
     for (id <TyphoonParameterInjection> parameter in _injectedParameters) {
-        hash = (NSUInteger) ((5 << hash) - hash + [[parameter description] hash]);
+        hash = (NSUInteger) ((magicNumber << hash) - hash + [[parameter description] hash]);
```

Comment:
It's a good style to set a comprehended name to every numeric literal.
While reading a code it's easier to understand what does this literal mean. I gave it this name because it's a famous name of obscure literals.